test/attr/attr_iboutlet.swift

```diff
@@ -50,13 +50,12 @@ protocol CP2 : class { }
 
 class NonObjC {}
 
-// Check where @IBOutlet can occur
+// Ensure that only ObjC types work
 @objc class X {
   // Class type
   @IBOutlet var outlet2: X?
   @IBOutlet var outlet3: X!
 
-  @IBOutlet var outlet1a: NonObjC // expected-error{{@IBOutlet property cannot have non-'@objc' class type 'NonObjC'}} {{3-13=}}
```

Comment:
Why was this test removed? It seems like it's reasonable to still have it here as are the other examples removed.

Reply:
Sorry but I don't think I follow - all of the "non-optional" removed tests produced different diagnostic messages expect to `P1.Type` `CP1.Type` `OP1.Type` because they are all metatype type, so I guess we can just keep one of them. I think the point of having all this different tests was to make examples as exhaustive as possible for all of the syntactic forms we have.

Reply:
So I looked at the code further. I don't think any regressions in test coverage has occurred.

Here is what is going on: despite what looks like different diagnostics in the test, the *same* diagnostic is actually being produced. The problem is that `expected-error` only does *sub*string matching, not equality checking, therefore if the compiler outputs "the green apple" and the test has `expected-error {{the green}}` then the test will pass.

I think we're fine here.

Reply:
I'm not trying to argue or anything, just trying to understand - tests have different substrings: `non-'@objc' class type`, `non-'@objc' protocol type` and `non-object type`.

Reply:
If you are confident, it's fine.